### PR TITLE
Added Default Image URL Feature

### PR DESCRIPTION
**Pull request by Biostate**
I added defaultImageUrl method to show placeholders if model doesn't have any media.

- [x] Changes have been thoroughly tested to not break existing functionality.
- [x] New functionality has been documented or existing documentation has been updated to reflect changes.

### CODE REVIEW

File: packages/spatie-laravel-media-library-plugin/src/Tables/Columns/SpatieMediaLibraryImageColumn.php [hunk 1]
```diff
@@ -67,7 +76,7 @@ public function getImagePath(): ?string
             return $state;
         }
 
-        return $record->getFirstMediaUrl($this->getCollection(), $this->getConversion());
+        return  $record->getFirstMediaUrl($this->getCollection(), $this->getConversion()) ?? $this->defaultImageUrl;
```
Comment (zepfietje) [inline packages/spatie-laravel-media-library-plugin/src/Tables/Columns/SpatieMediaLibraryImageColumn.php]:
```suggestion
        return $record->getFirstMediaUrl($this->getCollection(), $this->getConversion()) ?? $this->defaultImageUrl;
```

**Comment by danharrin** [comment]
Maybe this belongs on the base `ImageColumn` instead? Then use the default in the `getImagePath()` of both classes

File: packages/spatie-laravel-media-library-plugin/src/Tables/Columns/SpatieMediaLibraryImageColumn.php [hunk 2]
```diff
@@ -37,6 +39,13 @@ public function getConversion(): string
         return $this->conversion ?? '';
     }
 
+    public function defaultImageUrl(string $url): static
```
Comment (danharrin) [inline packages/spatie-laravel-media-library-plugin/src/Tables/Columns/SpatieMediaLibraryImageColumn.php]:
```suggestion
    public function defaultImageUrl(?string $url): static
```

**Comment by danharrin** [comment]
The docs should also be moved out of the plugin and into the main image column docs

**Comment by zepfietje** [comment]
Agree.

Mark as ready for review when you've implemented the requested changes, @Biostate! 😊

**Comment by Biostate** [comment]
I moved the defaultImageUrl to base ImageColumn class then i updated both SpatieMediaLibraryImageColumn and ImageColumn 's getImagePath methods.

**Comment by danharrin** [comment]
Thanks

**Pull request merged**